### DISCUSSION TITLE
Why controller based authorization got removed the docs?

### DISCUSSION THREAD

**Discussion by yaayes** [laravel/framework#52874] [General]
Hello everyone,

I started lately with L11, and I noticed that Controller based authorization isn't documented anymore, I've seen the changelog mention that you need to add the `AuthorizesRequests` manually, but I was surprised that methods in that trait especially `authorizeResource` isn't mentioned in the Authorization/Policy section. Is it going to be removed in the upcoming release?

**Comment by chuckrincon** [comment]
Hello.

Those authorization functions were moved to the official fortify and jetstream packages.
So they are not going to be removed in the next release.

Although you can create your own authentication and do it manually as mentioned in the documentation.

These packages were made for quick deployment and avoid user reinventing the wheel.

**Reply by yaayes** [reply]
Hello, Thanks for the answer but I think we're talking about different things, I'm talking about Authorization using Policies as mentioned in the Laravel v10 docs: https://laravel.com/docs/10.x/authorization#via-controller-helpers
